contrib/libcityhash/src/city.cc

```diff
@@ -115,7 +115,7 @@ static uint64 ShiftMix(uint64 val) {
 }
 
 static uint64 HashLen16(uint64 u, uint64 v) {
-  return Hash128to64(uint128(u, v));
+  return DB::Hash128to64(uint128(u, v));
```

Comment:
I did it this way because `#include <nmmintrin.h>` in the middle of the file will split the namespace in two, but I'll redo it with two namespaces now.